src/Microsoft.Framework.PackageManager/List/LibraryDependencyFlatRenderer.cs

```diff
@@ -89,7 +89,7 @@ public IEnumerable<string> GetRenderContent(IGraphNode<LibraryDescription> root)
 
                 if (!_hideDependent)
                 {
-                    var dependents = string.Join(", ", dependenciesMap[description].Select(dep => dep.ToString()).OrderBy(name => name));
+                    var dependents = string.Join(", ", dependenciesMap[description].Select(dep => dep.Identity.ToString()).OrderBy(name => name));
```

Comment:
Yup. But lets finish the analyzing first. The original source of all the LibraryDescriptions is ApplicaitonHostContext.DependencyWalker https://github.com/aspnet/dnx/blob/dev/src/Microsoft.Framework.PackageManager/List/DependencyListOperation.cs#L31. Any chance it returns any LibraryDescription with null Identity?